test/tests/concrete_ASR_validation/tests

```diff
@@ -1,55 +1,49 @@
-  [../]
-
-  [./ASRtest1_case2]
-    type = 'Exodiff'
-    input = 'asr_validation.i'
-    exodiff = 'asr_validation_case2_out.e'
-    custom_cmp = 'asr_validation.cmp'
-    cli_args = 'Outputs/file_base=asr_validation_case2_out Materials/ASR_expansion/max_volumetric_expansion=0.76e-3 Materials/ASR_expansion/characteristic_time=66.84 Materials/ASR_expansion/latency_time=-126.1'
-  [../]
-
-  [./ASRtest2_case2]
-    type = 'CSVDiff'
-    input = 'asr_validation.i'
-    csvdiff = 'asr_validation_case2_out.csv'
-    abs_zero = 1e-8
-    cli_args = 'Outputs/file_base=asr_validation_case2_out Materials/ASR_expansion/max_volumetric_expansion=0.76e-3 Materials/ASR_expansion/characteristic_time=66.84 Materials/ASR_expansion/latency_time=-126.1'
-  [../]
-
-  [./ASRtest1_case3]
-    type = 'Exodiff'
-    input = 'asr_validation.i'
-    exodiff = 'asr_validation_case3_out.e'
-    custom_cmp = 'asr_validation.cmp'
-    cli_args = 'Outputs/file_base=asr_validation_case3_out Materials/ASR_expansion/max_volumetric_expansion=0.38e-3 Materials/ASR_expansion/characteristic_time=17.24 Materials/ASR_expansion/latency_time=-2.55'
-  [../]
-
-  [./ASRtest2_case3]
-    type = 'CSVDiff'
-    input = 'asr_validation.i'
-    csvdiff = 'asr_validation_case3_out.csv'
-    abs_zero = 1e-8
-    cli_args = 'Outputs/file_base=asr_validation_case3_out Materials/ASR_expansion/max_volumetric_expansion=0.38e-3 Materials/ASR_expansion/characteristic_time=17.24 Materials/ASR_expansion/latency_time=-2.55'
-  [../]
+    [case2]
+      type = 'Exodiff'
+      input = 'asr_validation.i'
+      exodiff = 'asr_validation_case2_out.e'
+      custom_cmp = 'asr_validation.cmp'
+      cli_args = 'Outputs/file_base=asr_validation_case2_out '
+                 'Materials/ASR_expansion/max_volumetric_expansion=0.76e-3 '
+                 'Materials/ASR_expansion/characteristic_time=66.84 '
+                 'Materials/ASR_expansion/latency_time=-126.1'
+      detail = 'a maximum volumetric expansion of 0.76e-3, characteristic time of 66.84, and latency '
+               'time of -126.1, '
+    []
 
+    [case3]
+      type = 'Exodiff'
+      input = 'asr_validation.i'
+      exodiff = 'asr_validation_case3_out.e'
+      custom_cmp = 'asr_validation.cmp'
+      cli_args = 'Outputs/file_base=asr_validation_case3_out '
+                 'Materials/ASR_expansion/max_volumetric_expansion=0.38e-3 '
+                 'Materials/ASR_expansion/characteristic_time=17.24 '
+                 'Materials/ASR_expansion/latency_time=-2.55'
+      detail = 'and a maximum volumetric expansion of 0.38e-3, characteristic time of 17.24, and '
+               'latency time of -2.55.'
+    []
+  []
```

Comment:
Are we only keeping one type of test here?

Reply:
I deleted the csv checks because all the postprocessor data goes to the .e file, so they were redundant.